compiler/damlc/daml-stdlib-src/DA/Internal/Template/Functions.daml

```diff
@@ -242,6 +248,21 @@ instance Eq AnyContractKey where
   AnyContractKey aAny aRep == AnyContractKey bAny bRep =
     eqAny aAny bAny && aRep == bRep
 
+
+instance Ord AnyTemplate where
+  AnyTemplate a <= AnyTemplate b = lteAny a b
```

Comment:
Can we just use `primitive @"BELessEq"` here and for all the other `Ord` instances? If the goal is to make use of these in maps, it makes a lot more sense to use the built-in order for all of them.

```suggestion
  (<=) = primitive @"BELessEq"
```

Reply:
I’m a bit worried that sorting by value before we sort by template id is going to cause a fair amount of confusion. We can’t change the order of those since they are in stable types.

Reply:
hm fair point, if we get the same in map, we probably should do that.

Reply:
fixed in 64cd504949da6ca0d38ce46ee97ab78d92bf2b30, what you suggested fails in the lf conversion due to the way we use those primitives but fixed it slightly differently.

Reply:
It fails? That's unexpected.

Reply:
```
error type checking value DA.Internal.Template.Functions.$c<=:
|  unknown value:   
|  518032f41fd0175461b35ae0c9691e08b4aea55e62915f8360af2cc7a1f2ba6c:GHC.Types:primitive
```
Haven’t looked too closely at it, I expect it probably ends up looking slightly different in typeclass methods which breaks the way we match on it.

Reply:
e.g. this is the `Ord Int` instance in the stdlib. What's different?

<img width="271" alt="image" src="https://user-images.githubusercontent.com/63245722/112302401-f5b00700-8c92-11eb-8315-6dfd2121ed45.png">

Reply:
duh, I forgot the `@` in front of the string. Fixed in 7eca99ddf95c8dee21c92cd66566086326ac97e2